Dockerfile

```diff
@@ -1,5 +1,6 @@
 # syntax=docker/dockerfile:1.3-labs
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+ENV CONTAINER=""
```

Comment:
Generic name with no value set. 

```suggestion
ENV MONGODB_ATLAS_USER_AGENT="atlascli-container"
```

Reply:
I agree that the naming of the var could be improved. I think setting a different value does not really make sense though, because the variable is used as a flag and the value is never used. I set it to an empty string only because env vars in Docker cannot be undefined. (At least I did not find a way)

Reply:
I updated the variable name to be more descriptive, lmk what you think

Reply:
My point is that this PR could be just an `MONGODB_ATLAS_USER_AGENT="atlascli-container"` 
Variable name cannot be changed in this case as it is comming from documented feature: https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-env-variables/

Reply:
Feel free to not use that feature (not blocking anything in this PR). 
My only strong opinion is to avoid empty variables. If you need flag like now:

```
MONGODB_ATLAS_RUNNING_FROM_CONTAINER=true
```

One could argue that it introduces some level of proliferation. 
We should likely look into what we want is the effect:

For example disabling atlas cli to run any external binaries 
```
MONGODB_ATLAS_DISABLE_EXTERNAL_COMMANDS=true
MONGODB_ATLAS_USER_AGENT="atlascli-container"
```

This is not only easier to implement and maintain. It does fit well with existing features and config.
Each of those ideally should be connected with the config and documented so users will be aware of the differences when running container etc.



CONTAINER or other values are subject

Reply:
MONGODB_ATLAS_USER_AGENT="atlascli-container" doesn't seem to change the UserAgent in my container. So I changed the approach to set default in profile and then replace in case the new env var is set to true as suggested